kgforge/specializations/stores/demo_store.py

```diff
@@ -139,6 +139,44 @@ def _initialize_service(
     ):
         return StoreLibrary()
 
+    def _register_many(self, resources: List[Resource], schema_id: str) -> None:
```

Comment:
wait, where these methods not here?

Reply:
never mind, they were part of the `not_supported()` methods in the archetype

Reply:
And that is one of the reasons why I find it best to have the not_supported() in the implementation, because you're not even aware of all that can be implemented otherwise